Picker/Picker.js

```diff
@@ -116,12 +176,20 @@ const PickerBase = kind({
 						{isFirst ? '' : values[value - 1]}
 					</div>
 				</PickerButtonItem>
-				<div className={activeClassName}>
+				<div
+					aria-valuetext={typeof valueText !== 'undefined' ? valueText : values[value]}
+					className={activeClassName}
+					id={id}
```

Comment:
Let's keep `id` on the root and have a derived id here (e.g. `${id}_value`) that is used by `aria-controls`. You might also want to create it in computed and only deconstruct it in render so the id lands on the root automatically.

Reply:
I fixed the id to be modified in computed.
Please tell me if this code is different from your intention.

Reply:
Slightly, yes. I meant something like:

```
computed: {
  valueId: ({id}) => `${id}_value`
}
```

And then only deconstruct `valueId` from `props` so `id` lands on the root node and you use `valueId` for `aria-controls` and this node's id.

Reply:
I changed the name of the id prop according to your opinion.
Thank you.